src/main/java/uk/gov/pay/adminusers/persistence/entity/UserEntity.java

```diff
@@ -167,7 +167,7 @@ public void setFeatures(String features) {
     }
 
     public List<RoleEntity> getRoles() {
-        return servicesRoles.isEmpty() ? newArrayList() : singletonList(servicesRoles.get(0).getRole());
+        return servicesRoles.isEmpty() ? emptyList() : singletonList(servicesRoles.get(0).getRole());
```

Comment:
I suppose we could also do:

```java
return servicesRoles.stream().findFirst().map(ServiceRoleEntity::getRole).map(Collections::singletonList).orElse(Collections.emptyList());
```

But just because we can, doesn’t mean we should…